new/install.sh

```diff
@@ -14,16 +14,19 @@ php $PATCHDEMO/wikis/$NAME/w/maintenance/install.php \
 --pass=patchdemo1 \
 "$WIKINAME" "Patch Demo"
 
+mkdir $PATCHDEMO/wikis/$NAME/w/settings.d
+
+echo 'foreach( glob( __DIR__ . "/settings.d/*.php" ) as $conffile ) { include_once $conffile; }' >> $PATCHDEMO/wikis/$NAME/w/LocalSettings.php
```

Comment:
This is the technique and structure that vagrant uses for additional config files.